R/xml_interventions_define.R

```diff
@@ -1,13 +1,13 @@
 ##  TODO Improve documentation
 ##' Adds vaccine intervention parameterisation to baseList
-##' @param baseList List with experiment data.
-##' @param vaccine_parameterization Parameterization list (see example below)
+##' @param baseXMLfile List with experiment data.
```

Comment:
`baseXMLfile` is only used when we are actually interacting with the filesystem; use `baseList`